oo-install/site_assets/oo-install-bootstrap.sh

```diff
@@ -61,13 +61,18 @@ elif [[ $RUBYVER == ruby\ 2\.* ]]
 then
   RUBYDIR=''
 fi
-GEM_PATH="${TMPDIR}INSTALLPKGNAME/vendor/bundle/ruby/${RUBYDIR}gems/"
+
+OOINSTALL_GEM_PATH="${TMPDIR}INSTALLPKGNAME/vendor/bundle/ruby/${RUBYDIR}gems/"
 RUBYLIB="${TMPDIR}INSTALLPKGNAME/lib:${TMPDIR}oo-install/vendor/bundle"
-for i in `ls $GEM_PATH`
+for i in `ls $OOINSTALL_GEM_PATH`
 do
-  RUBYLIB="${RUBYLIB}:${GEM_PATH}${i}/lib/"
+  RUBYLIB="${RUBYLIB}:${OOINSTALL_GEM_PATH%/}/${i}/lib/"
```

Comment:
@jrafanie I think we originally set RUBYLIB to each of the gem's /lib/ directory because the ruby does not search in subdirectories specified in the RUBYLIB variable. It will only search top-level directories for libaries, so we have to explicity define each directory we expect libraries to be.

You are right, we should also be looping through the original GEM_PATH along with OO_INSTALL_GEM_PATH, or maybe loop through the whole list once we combine them. I'll look into making this change.